spec/features/user/logged_in_user_to_github_spec.rb

```diff
@@ -24,6 +24,29 @@
     expect(page).to have_content("5 Recent Repos")
     
   end
+
```

Comment:
Created this test to see if the second token works. Will need to be made more dynamic but everything is passing

Reply:
Sounds good